tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
I'm running this with -c local and w/o a full host inventory file, so that group isn't available.